recipe/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "momepy" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/momepy-{{ version }}.tar.gz
+  sha256: 413ed9221dc4b2190621f8cb37789f09ea882f7e81dfbffd5731a22da6e61bab
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [py<310 or s390x]
+
+requirements:
+  host:
+    - python
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - pip
+    - wheel
+  run:
+    - python
+    - geopandas >=0.12.0
+    - libpysal >=4.6.0
+    - networkx >=2.7
+    - packaging
+    - pandas !=1.5.0
+    - shapely >=2
+    - tqdm >=4.63.0
+
+
+test:
+  source_files:
+    - ./momepy/tests
+  imports:
+    - momepy
+  commands:
+    - pip check
+    - pytest -v ./momepy/tests --ignore=momepy/tests/test_dimension.py --ignore=momepy/tests/test_distribution.py --ignore=momepy/tests/test_elements.py --ignore=momepy/tests/test_utils.py
```

Comment:
could you please add the reason why we are ignoring those test files?

Reply:
added some comments!